### PR TITLE
Add small size modifiers for text/search inputs

### PR DESCRIPTION
**Pull request by kwood15**
Please review - have added small modifiers for text & search inputs/have prioritised medium sizing first.

### CODE REVIEW

File: nebula-components/base/_base.scss [hunk 1]
```diff
@@ -0,0 +1,4 @@
+hr {
+  margin-top: ($nb-spacing-unit * 2);
+  border-color: $nb-color-grey-light;
+}
```
Comment (rbrtsmith) [inline nebula-components/base/_base.scss]: We probably want to take this away so it doesn't accidentally ship.  We will look at tidying up the demo pages once the navigation is done and we can do some routing.

File: nebula-components/components/_navbar.scss [hunk 2]
```diff
@@ -1,4 +1,4 @@
-$nb-navbar-item-spacing: $nb-spacing-unit !default;
+$nb-navbar-item-spacing: ≈ !default;
```
Comment (rbrtsmith) [inline nebula-components/components/_navbar.scss]: What's happening here?

File: nebula-components/components/_text-input.scss [hunk 3]
```diff
@@ -12,6 +12,10 @@
   }
 
   .#{$nb-namespace}c-text-input--textarea {
-    min-height: 10rem;
+    min-height: $nb-text-input-textarea-min-height;
```
Comment (rbrtsmith) [inline nebula-components/components/_text-input.scss:15]: Good :+1:

File: nebula-components/components/_text-input.scss [hunk 4]
```diff
@@ -12,6 +12,10 @@
...
+  }
+
+  &.#{$nb-namespace}c-text-input--sm {
+    padding: ($nb-text-input-padding / 2);
```
Comment (rbrtsmith) [inline nebula-components/components/_text-input.scss]: Can we pull this into a variable too?  Just like we have variants of `nb-spacing-unit`

File: nebula-components/main.scss [hunk 5]
```diff
@@ -1,8 +0,0 @@
-@import 'settings';
-@import 'tools';
-@import 'resets';
-@import 'base';
-@import 'objects';
-@import 'components';
-@import 'utilities';
-
```
Comment (rbrtsmith) [inline nebula-components/main.scss:8]: Good.  Nice to clean up and remove redundant code :+1:

**Review by rbrtsmith — CHANGES_REQUESTED** [review]

File: nebula-components/base/_base.scss (hunk 1 above)
Comment (kwood15) [reply nebula-components/base/_base.scss]: Done - will remove any tags

File: nebula-components/components/_navbar.scss (hunk 2 above)
Comment (kwood15) [reply nebula-components/components/_navbar.scss]: Webstorm is tired! Will revert..

File: nebula-components/components/_text-input.scss (hunk 4 above)
Comment (kwood15) [reply nebula-components/components/_text-input.scss]: Done

**Comment by kwood15** [comment]
All above fixed - and have changed the small input to match same as medium width as looks better

**Review by rbrtsmith — APPROVED** [review]
Nice work!

**Pull request merged**